common/frame.go

```diff
@@ -645,6 +645,37 @@ func (f *Frame) check(selector string, opts *FrameCheckOptions) error {
 	return nil
 }
 
+func (f *Frame) setChecked(selector string, checked bool, opts *FrameCheckOptions) error {
+	setChecked := func(apiCtx context.Context, handle *ElementHandle, p *Position) (any, error) {
+		return nil, handle.setChecked(apiCtx, checked, p)
+	}
+	act := f.newPointerAction(
+		selector, DOMElementStateAttached, opts.Strict, setChecked, &opts.ElementHandleBasePointerOptions,
+	)
+	if _, err := call(f.ctx, act, opts.Timeout); err != nil {
+		return errorFromDOMError(err)
+	}
+
+	return nil
+}
+
+// SetChecked sets the checked state of the first element found that matches the selector.
+func (f *Frame) SetChecked(selector string, checked bool, opts sobek.Value) error {
+	f.log.Debugf("Frame:SetChecked", "fid:%s furl:%q sel:%q", f.ID(), f.URL(), selector)
+
+	popts := NewFrameCheckOptions(f.defaultTimeout())
```

Comment:
Should we create a new options type for `setCheck`?

Reply:
I considered that but couldn't see any significant differences between `check`, `uncheck`, and `setChecked`. I decided to omit adding another option type to avoid unnecessary redundancy since all actions fall under the category of "check" actions.